tools/cassandra/cqlclient.go

```diff
@@ -106,14 +106,21 @@ const (
 		`WITH replication = { 'class' : 'SimpleStrategy', 'replication_factor' : %v};`
 )
 
+// NewCQLClient returns a new instance of CQLClient
+func NewCQLClient(hostsCsv string, port int, user, password, keyspace string) (CQLClient, error) {
```

Comment:
don't think you need this public method anymore. Your CheckCompatibleVersion() can just use the package private newCQLClient()